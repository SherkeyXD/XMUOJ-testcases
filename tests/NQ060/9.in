1190782797 536
0 3410 434188 759240 788006 864472 1025410 1547000 1611600 1616076 2052886 2313608 2496300 2545222 2557118 2706757 3019100 3194504 3423420 3492720 3525705 3700180 3825409 3861000 4153320 4266864 4367776 4828560 5420166 5549068 5830748 5911299 6288002 6437211 6499584 7154406 7940383 8625456 8647884 8697150 8774826 9023562 9080864 9612008 9926398 11618625 12063285 14272006 14346800 14685043 15369606 15545172 16002792 16116300 16455383 17398055 18028075 18851623 18983634 19173573 19340048 19474160 19540808 19676514 20099649 20334132 20546064 20911600 22473288 22718223 23948820 24800080 25188300 25640520 25944880 26939460 28431002 29455860 29689615 29713110 29717600 29755700 31351083 31769140 33632307 34675950 35054405 35066665 35532072 36948380 37002208 37092384 37170354 38060568 38258539 38333680 39610560 40383966 40730853 41252654 42238676 42519162 42916596 42938815 43966224 44696587 44732894 45560130 46619664 46671885 46901163 47423388 47810620 48736748 48940200 49956075 52215361 52225960 52444370 53808867 54812160 55807992 58071090 58686550 59995200 61038273 62765581 62863593 63019659 63839097 63984824 64145506 64346832 64355610 65127056 67460120 67561896 68711247 68908416 69697576 69857136 70137465 70547000 71220474 74526926 75250560 76949158 77240280 78450138 78478914 79115696 80079619 81587800 82578798 84490832 84618260 87949442 88488477 89213952 89247744 90712632 91044993 91507040 92699200 93275974 93811632 94805592 95751799 96289695 99078538 100377900 101649930 101936122 102360313 105305688 105714705 106422750 106839054 107003610 107929179 108588408 110435677 111093345 111365530 112220445 113159134 113232220 116017576 117744246 120687360 121300480 121306582 121429760 122890284 123344439 124221636 124415574 125575716 125729828 126051030 126163647 126184880 126953996 127072605 127924840 128289198 129471340 130049552 131692704 134057556 134455608 134707604 134774676 134911235 136387044 136539111 136661200 137156120 139257768 140067096 141638780 143035591 143674902 144037944 148211105 148887970 149688546 150478490 151745815 153262854 155989962 156535224 156792104 158306885 158382420 159737230 161520520 161915358 163702902 165798110 166779609 168252672 170656000 172585296 173570979 175192312 175289655 175295330 175432431 176800312 178279872 178516139 178901796 179806880 180538771 182847240 183635736 184874864 185250716 187163972 192170119 193290952 193484454 195523510 196840080 198327530 198419976 199125399 199726695 199734561 200297808 201070998 204709457 204774722 205670760 206656008 207140586 207478725 208588419 209754725 213519072 213659719 214722970 217078554 219377808 220615738 222443460 223159248 223664113 226489152 228224104 228956532 230254358 230836815 233446000 234728451 235212390 235459812 236198234 238257398 242278050 243975744 244894050 247160322 247263443 250645512 251316426 254807398 258316884 258908300 259488856 259789257 260878860 261119040 261129024 267697560 267764574 269406293 274782470 276164811 276933248 280858950 281972704 282524088 283098130 284590540 284994852 287690273 288109812 291452358 296366798 296546292 296553978 298156552 300632639 301203873 302421636 304821784 305387677 306045348 306483948 307059480 307907376 308675360 308689710 308962500 308970834 310633650 311136552 311738690 312430995 313867872 314115784 314627232 315114867 315415750 316381297 316541361 318641730 325105257 329235139 329750758 335701141 336609876 340008091 340638408 342127006 342816543 343210920 344826011 345388398 346103885 346752960 348096173 349525328 349617156 349780600 352120337 354043600 356635760 356901496 357009975 357523056 358764280 360847835 361782270 363422952 364710221 364891680 365837436 367601640 368919456 371074400 372839052 375181420 375914531 376866976 377033930 382295915 383255343 387463371 389635206 389822400 390582785 392031484 393048632 393398592 397164060 399493444 406990462 408852693 410903685 418693650 419114762 421410144 426732192 431473424 438157776 446214201 455191914 458198104 462931390 463837472 464016777 464095134 464320441 465725700 466670400 467681373 468171543 468644182 476222177 479204976 485277912 486255744 496549700 496871430 499955836 500005122 500144400 501882030 503744325 505469692 507611712 508416516 509275585 510368664 522426480 524100155 525157723 526902440 532688800 535716951 535990654 541256245 543975698 547509600 548940816 549203460 562094024 562836664 564773619 567065160 567113170 569078196 569630540 569784188 574670096 575895267 577877860 583852692 592192752 608805315 620897508 622844281 628910900 631893828 633244170 637228128 641800220 642411396 649645200 652859548 652899943 660707835 668294528 675793248 675888600 676672080 678857388 683552358 684510398 686961259 690398024 697081770 699369575 701168064 703834746 707201856 709904394 715246228 717500130 722361972 726365922 726749280 727694324 731409180 731952459 732358824 739048190 741879705 742408953 744956408 750374966 752599356 763369992 773046920 776689183 789491363 790201442 803855884 808800195 825791170 861390306 882885375 906092388 907021441 918604965 919838870 928857452 934999680 948066912 973595412 974643390 998703036 1023490986 