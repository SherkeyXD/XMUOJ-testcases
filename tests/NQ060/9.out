2545222 37170354 419114762 731952459
3423420 106839054 161915358 918604965
11618625 126184880 505469692 547509600
175192312 207478725 258908300 549203460
